src/button-icons/buttons-icons.stories.html

```diff
@@ -0,0 +1,83 @@
+<article>
+  <h2>Button icons</h2>
+  <div>
+    <table>
+      <tr>
+        <div>
```

Comment:
nit: this div in between the `<tr>` and the `<td>` is sorta weird. There shouldn't be a need for it. Does something break if it's removed?

Reply:
I removed it and it works fine.